assets/sections.js

```diff
@@ -2,203 +2,127 @@ angular.module('cafapi.templates', []).run(['$templateCache', function($template
   'use strict';
```

Comment:
Comment included above.

The javascript changes are automatically generated when you run grunt, its for preloading pages for performance reasons.

I asked about this myself and was told to check in in for the reason above